press/press/doctype/site/site.py

```diff
@@ -2642,6 +2659,54 @@ def get_database_performance_report(self):
 		result["is_performance_schema_enabled"] = is_performance_schema_enabled
 		return result
 
+	def check_if_disk_usage_exceeded(self, save=True):
+		if self.free:
```

Comment:
this only skips for free sites not free teams. We should consider free teams as well?